fe/src/main/java/org/apache/doris/analysis/CreateMaterializedViewStmt.java

```diff
@@ -187,6 +197,14 @@ else if (functionChild0 instanceof CastExpr
                     throw new AnalysisException("The children of aggregate function only support one original column. "
                                                         + "Error function: " + functionCallExpr.toSqlImpl());
                 }
+
+                if (functionName.equalsIgnoreCase("bitmap_union")) {
+                    if (!slotRef.type.isIntegerType()) {
```

Comment:
The child type of bitmap_union must be bitmap rather then integer.
Also the child type of to_bitmap function has been checked in analyzer.